### ISSUE TITLE
allow seperate drpm repos

### ISSUE THREAD

**Issue by nirik** [rpm-software-management/dnf5#408]
I am not sure if this would best be filed here or libdnf or librepo or what, but I will start here. Feel free to move. :)

Right now, drpms are normally created by createrepo_c at repodata creation time. A prestodelta.xml file is added to the repodata and a drpms/ directory is made at the top level along side of the repodata directory. 

However, it would be much better for Fedora at least (and likely others) if it would be possible to create repos that are _just_ drpms. ie, just have a drpms/ and repodata/prestodelta.xml (or whatever). This would allow creating drpms out of band and populating it when desired instead of having it tied to repo creation time. 

So, the request here would be to allow having a drpms only repo and have dnf check for drpms for all the _other_ enabled repos in those drpm only repos. 

Hopefully this is possible. I understand if it's not a high priority. :)

**Comment by nirik** [comment]
Yeah, drpms are a trade-off. They make sense when you have very limited bandwith/download ability, but have lots of CPU cycles and disk space. These days, I think many people have a reasonably fast connection, so the trade off is not worthwhile. 

I was asked to file and request this, so I did. ;)

**Comment by j-mracek** [comment]
Yes, but the trade is not efficient.

**Comment by j-mracek** [comment]
I am sorry but righ now we have too many request. Keeping the issue opened would be polity but not helping anyone, because there is very low chance that we will have a time to implement the new feature. I am closing it. Please feel free to reopen when you will find good reasons to increase priority of the request.

**Comment by nirik** [comment]
Yep. Thats completely fine. Thank you for your time.
